README.md

```diff
@@ -32,14 +26,17 @@ npm install --save @microsoft/atlas-css
 yarn add --save @microsoft/atlas-css
 ```
 
-You may access any file within the `/css` folder using the following UNPKG url. Just add the path to the file, relative to the at the end:
+You may access any file within the `/css` folder using the following UNPKG url. Just add the path to the file, relative to the at the end.
 
-```http
-https://unpkg.com/browse/@microsoft/atlas-css@0.0.1/
+```sh
+https://unpkg.com/browse/@microsoft/atlas-css/ # Will redirect to latest version
+https://unpkg.com/browse/@microsoft/atlas-css@<version>/ # Use this pattern on your page
 ```
 
 ## Contributing
 
+While this project is open source, it's primary purpose is to serve Microsoft web properties through a css-only implementation of a design system. We do love contributions to both our documentation (`/site` folder) and our framework (`/css`).
```

Comment:
"it's primary purpose" => "its primary purpose"?

Reply:
Pesky apostrophe removed!